source/main/index.js

```diff
@@ -104,7 +104,12 @@ const onAppReady = async () => {
 
   Logger.info('Updating System-info.json file', { ...systemInfo.data });
 
-  Logger.info('DEBUG PATH', { path: process.env.PATH });
+  // We need DAEDALUS_INSTALL_DIRECTORY in PATH
+  // in order for the cardano-launcher to find wallet and node bins
+  process.env.PATH = [
+    process.env.PATH,
+    process.env.DAEDALUS_INSTALL_DIRECTORY,
```

Comment:
This will work, but I think it would be better if the script which starts Electron sets the `PATH` variable. This script is created by the installer anyway. Then Daedalus doesn't need to have a `DAEDALUS_INSTALL_DIRECTORY` variable.